app/models/user_external_uuid.rb

```diff
@@ -0,0 +1,6 @@
+class UserExternalUuid < ActiveRecord::Base
+
+  belongs_to :user, inverse_of: :external_uuids
+
+  validates :uuid, :user, presence: true
```

Comment:
uniqueness too? (know it is also in db constraint)

Reply:
random separate question - in Tutor should we prepend the tool guid so that we can be more certain that the user "uuid" is actually unique?